cmip6_downscaling/methods/bcsd/tasks.py

```diff
@@ -133,7 +133,7 @@ def fit_and_predict(
     coarse_obs_full_time_ds = xr.open_zarr(coarse_obs_full_time_path)
     experiment_train_full_time_ds = xr.open_zarr(experiment_train_full_time_path)
     experiment_predict_full_time_ds = xr.open_zarr(experiment_predict_full_time_path)
-
+    experiment_train_full_time_ds['time'] = coarse_obs_full_time_ds.time.values
```

Comment:
Maybe we add a short note explaining why this is needed incase we see a similar error in the future.

Reply:
great point! i'll add a comment.

Reply:
added a comment - @norlandrhagen - any chance you'd like to tackle this TODO?